package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "forta-bot-challenge-1",
+  "displayName": "Forta Bot Challenge 1",
+  "version": "0.0.1",
+  "description": "Forta Agent for monitoring Nethermind bot deployments, updates, and deletions",
+  "longDescription": "Welcome to the Forta Bot Challenge 1 project!\n\nThis Forta bot is designed to monitor bot deployments, updates, and deletions carried out by Nethermind. The bot generates alerts whenever a bot is created, updated, or deleted, helping to ensure transparency and traceability of bot management activities.\n\nThe Forta Agent SDK and CLI tool offer a seamless experience for developers who wish to leverage the decentralized and trustless environment provided by the Forta Network. This project is an example of how you can build powerful and secure bots that interact with the Forta platform and take advantage of its cutting-edge capabilities.\n\nJoin the Forta Network community and learn more about our mission to revolutionize decentralized finance and blockchain analytics: https://forta.org/",
+  "repository": "https://github.com/jmt-genius/forta-challenge",
+  "licenseUrl": "https://github.com/jmt-genius/forta-challenge/blob/one/LICENSE.md",
+  "promoUrl": "https://forta.org",
+  "chainIds": [
+    1
+  ],
+  "scripts": {
+    "format": "prettier --write \"src/**/*.ts\"",
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "alert": "npm run build && forta-agent run --alert",
+    "sequence": "npm run build && forta-agent run --sequence",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "info": "forta-agent info",
+    "logs": "forta-agent logs",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "stake": "forta-agent stake",
+    "test": "jest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.1.48",
+    "forta-agent-tools": "^3.2.12"
```

Comment:
As in this bot `"forta-agent-tools"` is only used for testing, it can be moved to `devDependencies`